### DISCUSSION TITLE
filter by nested back relations

### DISCUSSION THREAD

**Discussion by gvulu** [pocketbase/pocketbase#5825] [Q&A]
Hi there,

Using as reference [working with relations](https://pocketbase.io/docs/working-with-relations/ ) and assuming there is **user_comments** field in posts and that there is another third multiple relation from comments to another table  subcomments via field user_subcomment, I would like to filter posts like: 

`
filter: 'comments_via_user_comments.subcomments_via_user_subcomment.message ~"something"' 
`

This does not work in my tests, any other way to achieve the same in a single call? 

Thanks!

**Comment by ganigeorgiev** [comment]
_Rant: I would appreciate if users stop posting vague questions like this. I consider it low effort and lazy. It is just frustrating reading it nonstop and I feel that if I leave the repo unmantained even for a week the issues and discussions tab will rot. In the beginning I was more welcoming to this type of posts but I no longer think that it is a good idea to ignore or encourage it and I will call it out directly, even if there is a risk users to get offended by it. My expectations are that everyone here is a developer, no matter whether that is frontend, backend, qa, devops, etc, and to be able to understand that posts like "it doesn't work" without details and steps to reproduce are not really helpful and most likely will get nowhere (I'll consider creating templates for this in the near future)._


In any case, it is not clear from your description what is your current collections structure, nor what is the desired behavior.

Instead of brushing it off that it doesn't work, try to understand why. It may as well be a PocketBase issue and if that turn out to be the case it would need to be fixed.

The above rule most likely doesn't work because of:
- the used collections and relation field names are incorrect
- you may need to use the `?~` operator for "any/at-least-one" of type of constraint
- there is a cache on frontend/reverse proxy level (very common if using Next.js)

If you are still not able to identify the culprit, please elaborate a little more on your environment (where your code is running, how you are sending the request), try to provide a more complete code sample with your actual call and export your collections configuration as json (Admin UI > Settings > Export collections).
